guides/common/modules/ref_facts-settings.adoc

```diff
@@ -15,10 +15,10 @@ The content of this fact should be the full label of the organization.
 | *Ignore facts for domain* | No | Stop updating domain values from facts.
 | *Update subnets from facts* | None | {Project} updates a host's subnet from its facts.
 | *Ignore interfaces facts for provisioning* | No | Stop updating IP and MAC address values from facts (affects all interfaces).
-| *Ignore interfaces with matching identifier* | [`lo`, `en*v*`, `usb*`, `vnet*`, `macvtap*`, `;vdsmdummy;`, `veth*`, `tap*`, `qbr*`, `qvb*`, `qvo*`, `qr-*`, `qg-*`, `vlinuxbr*`, `vovsbr*`, `br-int`] | Skip creating or updating host network interfaces objects with identifiers matching these values from incoming facts.
+| *Ignore interfaces with matching identifier* | [`lo`, `en*v*`, `usb*`, `vnet*`, `macvtap*`, `;vdsmdummy;`, `veth*`, `tap*`, `qbr*`, `qvb*`, `qvo*`, `qr-\*`, `qg-\*`, `vlinuxbr*`, `vovsbr*`, `br-int`] | Skip creating or updating host network interfaces objects with identifiers matching these values from incoming facts.
```

Comment:
For some reason, `qg-\*` renders with the backslash in the output on my computer. Can you please double check on yours if it's necessary?

Reply:
Hi @Lennonka , I 've checked this change in downstream and there it renders correctly. Actually it was of my highest interest to get it proceed correctly in Pantheon. So I've made this change on 6.11 branch downstream and the guide in Pantheon started building correctly. It was just a test as I knew it will be soon overwritten by the automatic sync from upstream. So after the sync, the Administration guide stopped building again. So to summarize, in downstream, it renders correctly just with this syntax. Wasn't able to find any other syntax that would not break the build in Pantheon.

Reply:
Can you please check if it renders correctly in _upstream_ docs?

Reply:
Hi @Lennonka , @maximiliankolb in upstream docs the first part renders correctly, but the second has a backslash.
Like this: qr-*, qg-\*
And should be: qr-*, qg-*
In dowstream docs, both render correctly: qr-*, qg-*
Not sure why it is and what to do with it.
The only certain thing is that if we don't correct it, the build doesn't proceed in downstream and we can't publish the whole guide for 6.11. 
Lena, please could you help to figure out how to make this rendering correctly in both upstream and downstream?
Thank you!

Reply:
@maximiliankolb okay, perfect!  I will use the condition for Satellite then. As i really need to get the rendering resolved for downstream. Thank you!

Reply:
Hi @maximiliankolb , I've discovered another problem. Actually, I think the syntax might be wrong in the upstream docs. 
I'm checking the branch 3.1 (as Satellite 6.11 docs are generated from it) and master.
On both branches, there is this in the ref_fact-settings.adoc file: `qr-*`, `qg-*`,
However, if you check Foreman docs, that * sign, which is causing the issue with downstream build is not visible anyway - please check https://docs.theforeman.org/3.1/Administering_Red_Hat_Satellite/index-foreman-el.html#facts_settings_admin and https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#facts_settings_admin
It's rendered just as qr- and qg-.
The question is whether it's correct like this or it should be displayed as qr-* and qg-*?

Reply:
I see what you mean. I have no idea, but I assume it should be displayed as expected from the source file. Maybe a backslash fixes the issue?

While you're at it: Please remove `satellite` and use an attribute here: `Exclude pattern for facts stored in satellite`.

Reply:
Hi @maximiliankolb , apologies, but the backslashes is what I suggested by my commit. But you didn't want to accept it. I believe the backlash is the correct Asciidoc syntax here.  I really don't understand why in upstream, it does work for the first occurrence, but doesn't work for the second one (qg-* is not rendered correctly). In downstream, the backlashes fix both occurrences (I've checked by a commit to the downstream repo). However, this was corrected just temporarily as it was overwritten by upstream branch 3.1 automatically.  @Lennonka , please could you try to help with the correct syntax here? Apologies, I've ran out of ideas and am flooded with other tasks. Thank you!

Reply:
Hi @mdolezelova 

I think using a single backslash works: https://github.com/theforeman/foreman-documentation/pull/1315

You can update your PR and then check if that works for you downstream. If it improves upstream docs, I will happily merge of course.